.rubocop_todo.yml

```diff
@@ -367,7 +367,7 @@ Style/SignalException:
 
 # Offense count: 137
 # Cop supports --auto-correct.
-Style/SingleSpaceBeforeFirstArg:
+Style/SpaceBeforeFirstArg:
```

Comment:
Sure thing. I guess this crept in to silence some obnoxious warnings in my editor.